packages/blockchain-wallet-v4-frontend/src/layouts/Wallet/MenuLeft/Navigation/template.tsx

```diff
@@ -94,9 +84,10 @@ const Navigation = (props: OwnProps & Props) => {
       </LinkContainer>
       {values(
         mapObjIndexed(
-          (coin: SupportedCoinType, i) =>
+          (coin: SupportedCoinType | SupportedFiatType, i) =>
```

Comment:
this is used on 4 places we might make one common type and use it.

Reply:
Agree, maybe I call it `SupportedWalletCurrencyType` ?